### DISCUSSION TITLE
github student developer registration does not acept nepali date?

### DISCUSSION THREAD

**Discussion by ishwor12** [community/community#21263] [GitHub Education]
can anyone answer to this and can it be done anything

**Comment by mariadhari6** [comment]
Hi @ishwor12 
I think, to be eligible for the GitHub Student Developer Pack, you must:
- Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution
- Have a verifiable school-issued email address or upload documents that prove your current student status
